system-contracts/bootloader/tests/bootloader/bootloader_test.yul

```diff
@@ -1,65 +1,154 @@
+ }
+
+ function TEST_ceilDiv() {
+     testing_assertEq(ceilDiv(1, 0), 0, "Dividing by 0")
+     testing_assertEq(ceilDiv(0, 1), 0, "Dividing with overflow")
+     testing_assertEq(ceilDiv(5, 2), 3, "Invalid division")
+     testing_assertEq(ceilDiv(6, 2), 3, "Invalid division")
+ }
+
+ function TEST_lengthRoundedByWords() {
+     testing_assertEq(lengthRoundedByWords(65), 96, "Invalid word length")
+ }
+
+ function TEST_getGasPrice() {
+     let baseFee := basefee()
+     testing_log("Base Fee", baseFee)
+     testing_assertEq(getGasPrice(6, 5), baseFee, "Invalid gas price")
+ }
+
+//function TEST_getGasPrice_maxPriorityFeeGreaterThenMaxFee() {
+//    testing_testWillFailWith(13)
+//    getGasPrice(5, 6)
+//}
+
+//function TEST_getGasPrice_baseFeeGreaterThenMaxFee() {
+//    testing_testWillFailWith("Base fee greater than max fee")
+//    let baseFee := basefee()
+//    getGasPrice(baseFee, baseFee)
+//}
+
+function TEST_getOperatorRefundForTx() {
+    let transactionIndex := 10
+
+    let expected := 3872
+
+    testing_assertEq(getOperatorRefundForTx(transactionIndex), mload(expected), "Invalid refound for tx")
 }
 
-function TEST_safeDiv() {
-    testing_assertEq(safeDiv(4, 2, "err"), 2, "Simple division")
-    testing_assertEq(safeDiv(5, 2, "err"), 2, "Rounding")
-    testing_assertEq(safeDiv(5, 3, "err"), 1, "Rounding down")
-    testing_assertEq(safeDiv(4, 3, "err"), 1, "Rounding down")
-    testing_assertEq(safeDiv(0, 3, "err"), 0, "Rounding down")
-}
-function TEST_safeDivAssert() {
-    testing_testWillFailWith("divByZero")
-    safeDiv(4, 0, "divByZero")
+function TEST_getOperatorOverheadForTx() {
+    let transactionIndex := 10
+
+    let expected := 323872
+
+    testing_assertEq(getOperatorOverheadForTx(transactionIndex), mload(expected), "Invalid operator overhead for tx")
 }
 
-function TEST_asserts() {
-    testing_testWillFailWith("willFail")
-    safeSub(10, 12, "willFail")
+function TEST_getOperatorTrustedGasLimitForTx() {
+    let transactionIndex := 10
+
+    let expected := 643872
+
+    assertEq(getOperatorTrustedGasLimitForTx(transactionIndex), mload(expected), "Invalid trusted gas limit for tx")
 }
 
-function TEST_safeMul() {
-    testing_assertEq(safeMul(4, 2, "err"), 8, "Simple")
-    testing_assertEq(safeMul(0, 2, "err"), 0, "With zero")
-    testing_assertEq(safeMul(0, 0, "err"), 0, "With zero")
-    testing_assertEq(safeMul(2, 0, "err"), 0, "With zero")
+function TEST_getCurrentCompressedBytecodeHash() {
+    let pointer := mload(COMPRESSED_BYTECODES_BEGIN_BYTE())
+    let expected := add(COMPRESSED_BYTECODES_BEGIN_BYTE(), pointer)
+
+    testing_assertEq(getCurrentCompressedBytecodeHash(), mload(expected), "Invalid current")
 }
 
-function TEST_safeMulAssert() {
-    testing_testWillFailWith("overflow")
-    let left := shl(129, 1)
-    testing_log("left", left)
-    safeMul(left, left, "overflow")
+function TEST_checkOffset_success() {
+    //offset value := 8534624
```

Comment:
?

Reply:
Seems like a very simple test that asserts the value of offset. I think it makes sense to keep it, although it's not required.

Reply:
I was mainly referring to the `//offset value := 8534624` comment, it seemed like a commented out code

Reply:
removed.